src/Handler/DispatchHandler.php

```diff
@@ -60,7 +60,12 @@ protected function dispatcher()
         return FastRoute\simpleDispatcher(function (RouteCollector $collector) {
             foreach ($this->directory as $request => $action) {
                 list($method, $path) = explode(' ', $request, 2);
-                $collector->addRoute($method, $path, $action);
+
+                $collector->addRoute(
+                    $method,
+                    $this->directory->prefix($path),
```

Comment:
I went back and forth on this... is it better to add the prefix to each dispatched URI, or better to strip it from the request URI.

Interested in thoughts either way.